airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1875,17 +1875,26 @@ components:
         dag_id:
           type: string
           readOnly: true
-        execution_date:
-          description: >
-            The execution date. This is the time when the DAG run should be started according
-            to the DAG definition.
+        schedule_date:
+          type: string
+          nullable: true
+          description: |
+            The logical date (schedule date, previously called execution date). This is the time or interval
+            covered by this DAG run, according to the DAG definition.
 
             The value of this field can be set only when creating the object. If you try to modify the
             field of an existing object, the request fails with an BAD_REQUEST error.
 
             This together with DAG_ID are a unique key.
+          format: date-time
+        execution_date:
           type: string
+          nullable: true
+          description: |
+            The execution date. This is the same as schedule_date, kept for backwards compatibility.
+            A request with both schedule_date and this field will fail with an BAD_REQUEST error.
```

Comment:
Will it fail when both are supplied or when they have different values?

Reply:
You got me here 😆 I was undecided between the two and ended up doing different things in the two places. I’ll change the API spec to match implementation for now (i.e. only fail if values differ).